src/Provider/Provider.Web/Configuration/ServiceClaims.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Esfa.Recruit.Provider.Web.Configuration
+{
+    public enum ServiceClaim
+    {
+        DAA = 4,
```

Comment:
might be worth increasing the numeric values like in reservations. Guards against having to change the values if a new Service claim added between `DAA` and `DAB for example